ko/template_extending/README.md

```diff
@@ -100,7 +100,7 @@
 {% endfor %}
 ```
 
-우리는 이 코드를 모든 컨텐츠 블록에 대한 템플릿의 일부로 거에요.
+우리는 이 코드를 모든 컨텐츠 블록에 대한 템플릿의 일부로 쓸 거에요.
```

Comment:
[Fix typo] 
Original Text: `We want to use this as part of our template for all the content blocks.`